codegen/debug.go

```diff
@@ -248,7 +257,7 @@ func NewDebugger(c *client.Client, w io.Writer, inputSteerer *InputSteerer, prom
 							return
 						}
 					}()
-				case "exit":
+				case "exit", "quit", "q":
```

Comment:
Nice, I was thinking about doing the same for `continue` (`c`)

Reply:
`case "continue", "c":` is already there :)